src/pages/storey.mdx

```diff
@@ -3,3 +3,24 @@ tags: ["storey"]
 ---
 
 # Introduction
+
+[Storey](https://docs.rs/cw-storey/latest/cw_storey/) is an alternative to
```

Comment:
Have we ever settled on whether we put links inline or split them up with the actual URLs at the bottom of the doc?

Reply:
I don't know, but what's the worry? That we'll have an increased maintenance burden if we're not consistent with this?

Reply:
No, no worries, just thought out loud about consistency